base/system/winlogon/lang/bg-BG.rc

```diff
@@ -1,4 +1,4 @@
-LANGUAGE LANG_BULGARIAN, SUBLANG_DEFAULT
+﻿LANGUAGE LANG_BULGARIAN, SUBLANG_DEFAULT
```

Comment:
As I see, there has been an invisible change at the very beginning of this file (and of all the others).
Please be sure you encode all these language resource files in **UTF-8 Without BOM** as they were originally.

Reply:
I see that notepad messes up the file when you copy/paste. :(

Reply:
Use notepad++, it has good settings about file encoding :)